posts/views.py

```diff
@@ -1,8 +1,18 @@
 from django.shortcuts import render
+from django.views.generic import ListView
+from .models import Post
 
 
-def feed(request):
-    return render(request, 'posts/feed.html', {'title': 'feed'})
+class PostListView(ListView):
+    model = Post
+    template_name = 'posts/feed.html'
+    context_object_name = 'posts'
+    ordering = ['-date_posted']  # order posts from newest to oldest
+
+    def get_context_data(self, **kwargs):
+        ctx = super(PostListView, self).get_context_data(**kwargs)
+        ctx['title'] = 'feed'
+        return ctx
```

Comment:
I don't think this is part of this PR, maybe you need to rebase?

Reply:
I fixed it :) @natifridman